src/openms/include/OpenMS/METADATA/MetaInfoRegistry.h

```diff
@@ -161,8 +163,8 @@ namespace OpenMS
 private:
     /// internal counter, that stores the next index to assign
     UInt next_index_;
-    using MapString2IndexType = std::map<String, UInt>;
-    using MapIndex2StringType = std::map<UInt, String>;
+    using MapString2IndexType = std::unordered_map<std::string, UInt>;
+    using MapIndex2StringType = std::unordered_map<UInt, std::string>;
```

Comment:
is this for performance?

Reply:
@timosachsenberg I can't recall making that change ;) Can you comment?
I would assume so.